test/bigtest/network/index.js

```diff
@@ -0,0 +1,26 @@
+import { camelize } from '@bigtest/mirage';
+
+// auto-import all mirage submodules
+const req = require.context('./', true, /\.js$/);
+const modules = req.keys().reduce((acc, modulePath) => {
```

Comment:
Could you please clarify, what does it mean 'acc'?

Reply:
Accumulator. This is a kind of boilerplate code that exists in every project where BigTest is set up, so I have not changed the any character of it. If you think of a better naming here, please suggest.